.github/PULL_REQUEST_TEMPLATE/bug.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Pull Request for a bug
+about: Use this template for bugs
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+<!-- Thank you so much for your time taking to contribute, your work is appreciated! 😃 -->
+
+## Description
+<!-- A few sentences describing the overall goals of the pull request' s commits. -->
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
+- [ ] Tests for the changes have been added
+- [ ] Docs have been added / updated
+
+## Issue
+
+**What is the current behavior?**
+<!-- You can also link to an existing issue here -->
+#<issue>
```

Comment:
this was positioned here because most of the times we fill this section as.. 
see issue 
or #xxxx  (directly duplicated the number of the issue)

So in order to avoid duplication the issue has been moved here. otherwise i propose to remove the 
**What is the current behavior** because it is written in the issue